{"uuid":"dfbf76b9-e5dd-4ecd-82f5-de58571733fe","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["38a80b4c-c004-44e6-b3b9-8573b83bec81"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673300042684,"stop":1673300042684}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673300042690,"stop":1673300042690},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673300043010,"stop":1673300043010},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673300043213,"stop":1673300043213},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"e2f69053-f364-4446-a8ad-1e02c4ca05bc-attachment","type":"image/png"},{"name":"Myscreenshot","source":"4d636d4f-1763-4323-b885-e026f48f5c3b-attachment"}],"parameters":[],"start":1673300043234,"stop":1673300043622}],"start":1673300042684,"stop":1673300043622}